downloads/pre-release/index.php

```diff
@@ -40,7 +40,7 @@
 <p><a href='<?=$helphost?>/version-history'>Keyman version history</a> (all products)</p>
 
 <?php
-  downloadSection('Keyman Desktop for Windows', 'windows', 'keymandesktop-$version.exe', 'beta alpha');
+  downloadSection('Keyman Desktop for Windows', 'windows', 'keyman-$version.exe', 'beta alpha');
```

Comment:
Until we get to 14.0 beta, won't the latest 13.0 beta be `keymandesktop-$version.exe`?

Reply:
Yeah but there is no current 13.0 beta so I don't think it matters.